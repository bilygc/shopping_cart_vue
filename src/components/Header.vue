<template>
    <div>
        <img alt="Vue logo" src="../assets/logo.png">
        <div id="nav">
        <router-link to="/">Home</router-link> |
        <router-link to="/about">About</router-link>
    </div>
    </div>
</template>
