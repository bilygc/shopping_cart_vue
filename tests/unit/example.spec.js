import { shallowMount } from '@vue/test-utils'
import Header from '@/components/Header.vue'

describe('Header.vue', () => {
  it('renders Header 1', () => {
    const msg = 'new message'
    const wrapper = shallowMount(Header, {
      stubs : ['router-link']
    })
    expect(wrapper.text()).toMatchSnapshot()
  })
  it('renders Header 2', () => {
    const msg = 'new message'
    const wrapper = shallowMount(Header, {
      stubs : ['router-link']
    })
    expect(wrapper.text()).toMatchSnapshot()
  })
})
