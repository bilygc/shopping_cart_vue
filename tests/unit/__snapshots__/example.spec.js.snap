// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.vue renders Header 1 1`] = `
"Home |
    About"
`;

exports[`Header.vue renders Header 2 1`] = `
"Home |
    About"
`;

exports[`Header.vue renders props.msg when passed 1`] = `
"Home |
    About"
`;
